### PR TITLE
Notes are created from completed tasks closes #95

### LINKED ISSUE #95: Generate new notes for completed todo list items
To make the todo list more useful, we should include an option to mark items as done, then upon clicking a button a user may prompt them to be turned into new notes and removed from the todo list. The todo item would be the title and the user may enter notes in the appropriate section.

### PR DESCRIPTION
**Pull request by Kekoa-Pickett**
Created "completedTasksToNotes()" that calls on the imported "createNoteFromTask(taskId, taskName)" and creates a note for each completed task. It then removes that task from the task list. Closes #95

### CODE REVIEW

**Review by thedonutdan — CHANGES_REQUESTED** [review]
This looks great! Lets get it integrated into our E2E testing though, make sure that when you mark a task as complete and press the add to notes button it creates new notes with the proper title.

**Comment by Kekoa-Pickett** [comment]
Hey, could you please tell me how you're getting that issue. On my end, I am seeing the completed tasks name become the new notes title when we click add to notes.

**Comment by thedonutdan** [comment]
> Hey, could you please tell me how you're getting that issue. On my end, I am seeing the completed tasks name become the new notes title when we click add to notes.

I'm not having any issues, my request was to incorporate checks into our E2E puppeteer testing to ensure we are testing that it works properly on the user's end. I believe the todo list tests are written by Vaibhav and the notes tests were mainly written by Joseph so if you need help get with them to get the tests written.

**Comment by Kekoa-Pickett** [comment]
Alright, the E2E test has been added and makes sure the correct name and info is added when adding a completed task.

File: source/js/__testing__/puppeteer/task->note.test.js
Comment (thedonutdan) [inline source/js/__testing__/puppeteer/task->note.test.js:1]: this file needs to be renamed. git has trouble tracking file names with a chevron in them

Comment (Kekoa-Pickett) [reply source/js/__testing__/puppeteer/task->note.test.js:1]: Sorry about that, just fixed it.

**Review by thedonutdan — APPROVED** [review]
Looks great!

**Pull request merged**